helm/cluster-api/templates/bootstrap/deployment.yaml

```diff
@@ -29,7 +29,7 @@ spec:
       containers:
       - args:
         - --metrics-bind-addr=0.0.0.0:{{ .Values.ports.metrics }}
-        - --feature-gates=MachinePool={{ .Values.featuregates.machinepool }}
+        - --feature-gates=MachinePool={{ .Values.featuregates.machinepool }},ClusterTopology={{ .Values.featuregates.clustertopology }}
```

Comment:
I'm not sure if bootstrap has this flag. I think I only had to enable it on control plane and core controller managers. I can check.

Reply:
It looks like it's not used in the bootstrap controller manager, but it looks like it will still start so I think this is okay.

Reply:
oh wow, I copied it from the manifests generated by `clusterctl`. If it's not used, I'll remove it.